src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {

```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line before the class declaration.